account_analytic_no_lines/tests/test_account_analytic_no_lines.py

```diff
@@ -0,0 +1,117 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import SavepointCase
+
+
+class TestAccountAnalyticNoLines(SavepointCase):
+
+    @classmethod
+    def setUpClass(cls):
```

Comment:
But this is not declared as `@classmethod`. That's the problem. See for example https://github.com/odoo/odoo/blob/06754b3237a44d8ba2a18bf407a3e65447ca390c/addons/account/tests/account_test_users.py#L8